### ISSUE TITLE
Node Exit Error, Unable to Restart

### ISSUE THREAD

**Issue by meta-lite** [iron-fish/ironfish#2060]
### What happened?

Left my node running overnight and came back to see the error listed below, attempting restart commands had no effect and reflected the same error message. 

### Version

v0.1.44

### Debug output

```shell
Iron Fish version               0.1.44 @ a79d445
Iron Fish library               0.0.21 @ a79d445
Operating system                Linux x64
CPU model(s)                    Intel(R) Core(TM) i5-10400 CPU @ 2.90GHz
CPU threads                     12
RAM total                       7.51 GiB
Heap total                      1.99 GiB
Node version                    v16.16.0
ironfish in PATH                true
Garbage Collector Exposed       false
Telemetry enabled               true
Node name                       Mycelium
Block graffiti                  Mycelium
Accounts head hash              0000000000029b3ef0fc5e5d24bc122385bca3c2dd7ace18b9fee8fe3d009727
Accounts head in chain          true
Accounts head sequence          142769
```

### Relevant log output

```shell
Version       0.1.44 @ a79d445
Node Name     Mycelium
Graffiti      Mycelium
Peer Identity KChEK76XO5Iti2o8GdOLHyWi3kfE4ZmYMr6a13m9X28=
Peer Agent    ironfish/0.1.44/a79d445
Peer Port     9033
Bootstrap     test.bn1.ironfish.network
 
WebSocket server started at :::9033
/home/host/.nvm/versions/node/v16.16.0/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:350
                        throw new Error('nullifierToNote mappings must have a corresponding noteToNullifier map');
                              ^

Error: nullifierToNote mappings must have a corresponding noteToNullifier map
    at /home/host/.nvm/versions/node/v16.16.0/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:350:31
    at async LevelupDatabase.withTransaction (/home/host/.nvm/versions/node/v16.16.0/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/storage/database/database.js:41:28)
    at async Accounts.removeTransaction (/home/host/.nvm/versions/node/v16.16.0/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:332:9)
    at async Accounts.expireTransactions (/home/host/.nvm/versions/node/v16.16.0/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:674:17)
    at async Accounts.eventLoop (/home/host/.nvm/versions/node/v16.16.0/lib/node_modules/ironfish/node_modules/@ironfish/sdk/build/src/account/accounts.js:157:9)
```

### Graffiti

Mycelium

**Comment by meta-lite** [comment]
Relaying my discord updates here: 
- was recommended to run a `accounts:rescan --reset` which failed on the first attempt
- ran a prune and chain download command 
- retried the `accounts:rescan --reset` and failed again. 
- Failure is approx halfway through the scans and displays the "killed" message

**Comment by hughy** [comment]
Errors like this one should be resolved by the Wallet 2.0 refactor released in [v0.1.48](https://github.com/iron-fish/ironfish/releases/tag/v0.1.48)

Please open a new issue if you run into this again!
